documentation/puppet_server_metrics_performance.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: "Applying metrics to improve performance"
+canonical: "/puppetserver/latest/puppet_server_metrics_performance.html"
+---
+
+[metrics]: ./puppet_server_metrics.markdown
+[tuning guide]: ./tuning_guide.markdown
+[metrics API]: ./metrics-api/v1/metrics_api.markdown
+[status API]: ./status-api/v1/services.markdown
+[developer dashboard]: #using-the-developer-dashboard
+[Graphite]: https://graphiteapp.org
+[Grafana]: http://grafana.org
+[sample Grafana dashboard]: ./sample-puppetserver-metrics-dashboard.json
+[`grafanadash`](https://forge.puppet.com/cprice404/grafanadash)
```

Comment:
Should this one and the following one have back-ticks around the words inside of the brackets?  That may be fine; I'm not really sure how the formatting works.